web/src/components/Layout.tsx

```diff
@@ -94,13 +94,13 @@ const Main = styled.main<{ fullWidth: boolean }>`
 `
 
 const Aside = styled.aside<{ languageSelectorHeight: number }>`
-  top: ${props => props.languageSelectorHeight + dimensions.headerHeightLarge}px;
+  top: ${props => props.languageSelectorHeight + dimensions.headerHeightLarge + dimensions.additionalSpacing}px;
   margin-top: ${props => props.languageSelectorHeight - dimensions.navigationMenuHeight}px;
   display: inline-block;
   position: sticky;
   width: ${dimensions.toolbarWidth}px;
   vertical-align: top;
-  transition: top 0.2s ease-in-out;
```

Comment:
With a 0s transition, you can completely remove the transition :)

Reply:
Which we shouldn't do, please readd. Except you have a reason for changing this?

Reply:
i was requested to remove the transition feature because when the page loads, the transition feature moves the toolbox down after some time.  This transition was unnecessary so i just set the time to 0 instead of removing it completely.

Reply:
That was me, I think it feels weird for the toolbar to float down after everything is done. Do you not?

Reply:
TBH I don't mind either, especially since I am not even sure when the transition happens/how it looks like, for some reason I can't reproduce it :D.
Then lets remove the transition completely then though.